athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/serde/v2/ListTablesResponseSerDe.java

```diff
@@ -93,7 +96,14 @@ protected FederationResponse doTypedDeserialize(JsonParser jparser, Deserializat
 
             String catalogName = getNextStringField(jparser, CATALOG_NAME_FIELD);
 
-            return new ListTablesResponse(catalogName, tablesList.build());
+            String nextToken = null;
```

Comment:
same comment here about checking the serizliation version and being stricter. curious if we should do that

Reply:
Added a `TODO` comment to enforce parsing when we upgraded to V3.